application/__tests__/example.macro.test.js

```diff
@@ -68,100 +82,39 @@ xdescribe('Single page example macro english html only', () => {
-  })
-})
-
-xdescribe('When a pattern has a nunjucks example', () => {
-  const parameters = { html: `${exampleId}.html`, nunjucks: `${exampleId}.njk` }
-  const document = documentFactory(parameters, options)
-
-  test('should have a button to show Nunjucks example', () => {
-    const codeExampleTabsContainer = document.getElementsByClassName('app-tabs')[0]
-
-    const exampleToggleLinks = codeExampleTabsContainer.querySelectorAll('li.app-tabs__item a')
-    expect(exampleToggleLinks.length).toBe(2)
-
-    const nunjucksExampleToggleLink = exampleToggleLinks[1]
-    expect(nunjucksExampleToggleLink.nodeName.toLowerCase()).toBe('a')
-    expect(nunjucksExampleToggleLink.getAttribute('href')).toBe(`#${exampleId}_nunjucks`)
-    expect(nunjucksExampleToggleLink.getAttribute('aria-controls')).toBe(`${exampleId}_nunjucks`)
-    expect(nunjucksExampleToggleLink.text).toBe('Nunjucks')
-  })
-
-  test(' Should include the Nunjcks macro code for the example\n\t\u001b[33;1mTODO: needs a better way to do the inclusion\u001b[0m', () => {
-    const exampleCode = document.getElementById(`${exampleId}_nunjucks`)
-    const fixtureCode = fs.readFileSync(path.join(fixturePath, 'test.njk')).toString()
-    expect(exampleCode.querySelector('pre code').innerHTML).toEqual(nunjucksEscape(fixtureCode))
-  })
-})
-
-xdescribe('When a pattern has Welsh content', () => {
-  const welshOptions = JSON.parse(JSON.stringify(options))
-  welshOptions.filters = Object.assign({}, options.filters)
-  welshOptions.globals = Object.assign({ hasWelsh: true }, welshOptions.globals)
-
-  const parameters = { html: `${exampleId}.html` }
-
-  const document = documentFactory(parameters, welshOptions)
-
-  test('Should include a language toggle for examples', () => {
-    const languageToggleLink = document.querySelector('a.language-toggle')
+  test('Should not have a language toggle for dual language examples', () => {
+    const welshParameters = { ...parameters, welsh: 'welsh' }
+    const welshDocument = documentFactory(welshParameters, options)
+    const languageToggleLink = welshDocument.querySelector('a[href="/design-library/new-tab-link/welsh/"]')
     expect(languageToggleLink).not.toBeNull()
   })
 
-  test('Should have a Welsh Language example version in a hidden div', () => {
-    const exampleHTMLCode = document.getElementById(`${exampleId}_welsh`)
-    expect(exampleHTMLCode.classList).toContain('govuk-visually-hidden')
-    expect(exampleHTMLCode.getAttribute('aria-hidden')).toBe('true')
-
-    const fixtureHTML = fs.readFileSync(path.join(fixturePath, 'test.cy.html')).toString()
-    expect(exampleHTMLCode.querySelector('pre code').innerHTML).toEqual(htmlEscape(fixtureHTML))
+  test('should have a button to show HTML code examples', () => {
+    const tabLink = document.querySelector('ul.app-tabs li.js-tabs__item a[href="#example-default-html"]')
+    const tabContentContainer = document.getElementById('example-default-html')
```

Comment:
Is it worth variablising `'example-default-html'` to show/ensure that it's the same value used multiple times?  (`aria-controls` is the other time).